.gitmodules

```diff
@@ -4,7 +4,7 @@
 	ignore = dirty
 [submodule "libvmi"]
 	path = libvmi
-	url = https://github.com/tklengyel/libvmi
+	url = https://github.com/libvmi/libvmi
```

Comment:
Please don't change the url here.

Reply:
I've pushed the latest master to `tklengyel/libvmi` you should be able to update the submodule to that.

Reply:
I just didn't know how to ask you to do this. Done.

Reply:
No worries